src/mmw/js/src/core/models.js

```diff
@@ -222,6 +222,14 @@ var TaskMessageViewModel = Backbone.Model.extend({
         this.set('iconClasses', 'fa fa-exclamation-triangle');
     },
 
+    setTimeout: function() {
```

Comment:
Consider renaming this to `setTimeoutError` or something similar to indicate it's an error state and that we aren't setting some timeout number/amount.